spec/integration/unhandled_errors_in_receivers_spec.rb

```diff
@@ -46,20 +49,67 @@
     expect(ExceptionThrowingReceiver.messages_received[1][2]).to eq(ExceptionThrowingReceiver.messages_received[0][2])
   end
 
+  it "crashes the receiver that received an error" do
+    Pwwka::Transmitter.send_message!({ sample: "payload", has: { deeply: true, nested: 4 }},
+                                     "pwwka.testing.foo")
+    allow_receivers_to_process_queues
+
+    expect(@testing_setup.threads[ExceptionThrowingReceiver].alive?).to eq(false)
```

Comment:
We're using Threads to simulate processes.  If the Thread is no longer alive, that means whatever it was running as stopped.  That's what we're testing for.